store/project.js

```diff
@@ -102,11 +115,23 @@ export const mutations = {
       id: `row-${Date.now()}`,
       zones
     }
-    const rows = [...state.project.rows, row]
-    state.project = { ...state.project, rows }
+    const desktop = {
+      ...state.project.desktop,
+      rows: [...state.project.desktop.rows, row]
+    }
+    const mobile = {
+      ...state.project.mobile,
+      rows: [...state.project.mobile.rows, row]
+    }
+
+    state.project.currentStep === stepsEnum.desktop
+      ? (state.project = { ...state.project, desktop })
+      : (state.project = { ...state.project, mobile })
```

Comment:
wow
take it easy

there is a  generic way to set such things
by passing `currentStep` as a key

plz, optimize lines 118 - 129